config/analytics.yml

```diff
@@ -1,3 +1,31 @@
 shared:
   candidates:
     - created_at
+  application_choices:
```

Comment:
What if there's an error here? A typo in a fieldname, or a field gets removed from the schema? 

Are there tests?

Reply:
Good question. I'll try and add

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5195/commits/23747c95af2b257f213d00dedc7ef1710e1e0c48